src/v2/view-builder/views/duo/ChallengeDuoAuthenticatorView.js

```diff
@@ -0,0 +1,35 @@
+import { loc } from 'okta';
+import Duo from 'duo';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+
+  className: 'duo-authenticator-challenge',
+  noButtonBar: true,
+
+  title  () {
+    return loc('oie.duo.verify.title', 'login');
+  },
+
+  postRender: function () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    // TODO Check contextualData.integrationType once we support other types than iframe
+    this.add('<iframe frameborder="0" title="' + this.title() + '"></iframe>');
+    Duo.init({
+      host: contextualData.host,
+      sig_request: contextualData.signedToken, // eslint-disable-line camelcase
+      iframe: this.$('iframe').get(0),
```

Comment:
Open question - how would this work if the Widget is embedded inside of another iframe? Would this pick up the parent frame or the Duo one as we'd like it to?

If possible, can we change this selector to lookup by `id` or `name`?